src/renderers/dom/client/syntheticEvents/__tests__/SyntheticEvent-test.js

```diff
@@ -81,13 +82,43 @@ describe('SyntheticEvent', function() {
     expect(syntheticEvent.target).toBe(null);
   });
 
+  it('should warn when accessing properties of a destructored synthetic event', function() {
+    spyOn(console, 'error');
+    var target = document.createElement('div');
+    var syntheticEvent = createEvent({srcElement: target});
+    syntheticEvent.destructor();
+    expect(syntheticEvent.type).toBe(null);
+    expect(console.error.calls.length).toBe(1);
+    expect(console.error.argsForCall[0][0]).toBe(
+      'Warning: This synthetic event is reused for performance reasons. If ' +
+      'you\'re seeing this, you\'re accessing a property on a ' +
+      'released/nullified synthetic event. This is set to null. See ' +
+      'https://fb.me/react-event-pooling for more information.'
+    );
+  });
+
+  it('should warn when setting properties of a destructored synthetic event', function() {
+    spyOn(console, 'error');
+    var target = document.createElement('div');
+    var syntheticEvent = createEvent({srcElement: target});
+    syntheticEvent.destructor();
+    expect(syntheticEvent.type = 'MouseEvent').toBe('MouseEvent');
+    expect(console.error.calls.length).toBe(1);
+    expect(console.error.argsForCall[0][0]).toBe(
+      'Warning: This synthetic event is reused for performance reasons. If ' +
+      'you\'re seeing this, you\'re setting a property on a ' +
+      'released/nullified synthetic event. This is effectively a no-op. See ' +
+      'https://fb.me/react-event-pooling for more information.'
+    );
+  });
+
   it('should warn if the synthetic event has been released when calling `preventDefault`', function() {
     spyOn(console, 'error');
     var syntheticEvent = createEvent({});
     SyntheticEvent.release(syntheticEvent);
     syntheticEvent.preventDefault();
-    expect(console.error.calls.length).toBe(1);
-    expect(console.error.argsForCall[0][0]).toBe(
+    expect(console.error.calls.length).toBe(3); // once each for setting `defaultPrevented`, accessing `nativeEvent`, and accessing `preventDefault`
```

Comment:
When calling `preventDefault`, we set the `defaultPrevented` access the `nativeEvent` properties. This leads to three warnings even if the developer only called `preventDefault`.